src/audioWorklet/index.js

```diff
@@ -28,7 +31,7 @@ p5.prototype.registerMethod('init', function () {
   // use p5's preload system to load necessary AudioWorklet modules before setup()
   this._incrementPreload();
   const onWorkletModulesLoad = function () {
-    initializedAudioWorklets = true;
```

Comment:
If you're going to remove the `initializedAudioWorklets` variable, can you make sure that the
```
if (initializedAudioWorklets) return;
```
statement on line 25 isn't necessary and if so, remove it?